### PR TITLE
Handle in_tax as well as in_tax_children template conditions

### PR DESCRIPTION
**Pull request by dgwatkins**
https://onthegosystems.myjetbrains.com/youtrack/issue/wpmlcore-7674

### CODE REVIEW

**Comment by dgwatkins** [comment]
review please :-)
@OnTheGoSystems/compatibility 
@OnTheGoSystems/wpml

**Review by strategio — DISMISSED** [review]
+1 @dgwatkins

**Comment by dgwatkins** [comment]
@strategio I need to re-request your review because the data provider was incorrect.
A small change to be able to build on both Linux & Mac also slipped in.

**Review by strategio — DISMISSED** [review]

File: composer.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
   },
   "scripts": {
     "post-update-cmd": [
-      "sed -i 's/final public function add_group_control/public function add_group_control/g' ./vendor/elementor/elementor/includes/base/controls-stack.php"
+      "sed -i'' 's/final public function add_group_control/public function add_group_control/g' ./vendor/elementor/elementor/includes/base/controls-stack.php"
```
Comment (andreasciamanna) [inline composer.json:57]: @dgwatkins can you explain this change?

Comment (dgwatkins) [reply composer.json:57]: @andreasciamanna I didn't really mean to commit that change. It actually slipped in during a fixup commit.

However, the change is required to run unit tests on my local host (Mac)

Linux just takes `-i` with no arguments.
Mac takes `-i` with the extension of the bak file (ie: `.bak`).

This way of calling it `-i''` (with no spaces) plays well with both Mac & Linux versions of sed.

Comment (andreasciamanna) [reply composer.json:57]: Good to know!

It would be better if you could move this change in a separate commit where you explain the reason of the change.

**Comment by andreasciamanna** [comment]
Anyway, +1

**Review by andreasciamanna — DISMISSED** [review]

File: composer.json (hunk 1 above)
Comment (dgwatkins) [reply composer.json:57]: I see you approved, so I will make sure I issue a separate commit next time.

**Pull request merged**